README.md

```diff
@@ -0,0 +1,31 @@
+[![Build status](https://ci.appveyor.com/api/projects/status/qd3q5801n57givny?svg=true)](https://ci.appveyor.com/project/acamino/appointment-reminders-csharp)
+
+
+## Apointment Reminders in ASP.NET MVC
+
+This example demonstrates how to use Twilio to create automatic appointment reminders for your business users.
+
+## Running Locally
+
+To run this project locally you'll need to move/rename `AppointmentReminders.Web/Web.config.sample` to `AppointmentReminders.Web/Web.config`.
+
+You'll need to change these values for the ones you need.
+
+```xml
+<appSettings>
```

Comment:
Just to ease the development process, I don't want to expose my Twilio account information.